bitnami/aspnet-core/README.md

```diff
@@ -47,153 +47,148 @@ The command removes all the Kubernetes components associated with the chart and
+| `ingress.path`                  | Ingress path                                                                                               | `/`                      |
+| `ingress.pathType`              | Ingress path type                                                                                          | `ImplementationSpecific` |
+| `ingress.certManager`           | Set this to true in order to add the corresponding annotations for cert-manager                            | `false`                  |
+| `ingress.hostname`              | Default host for the ingress resource, a host pointing to this will be created                             | `aspnet-core.local`      |
+| `ingress.annotations`           | Ingress annotations done as key:value pairs                                                                | `{}`                     |
+| `ingress.tls`                   | Enable TLS configuration for the hostname defined at ingress.hostname parameter                            | `false`                  |
+| `ingress.secrets`               | If you're providing your own certificates, please use this to add the certificates as secrets              | `[]`                     |
+| `healthIngress.enabled`         | Enable healthIngress controller resource                                                                   | `false`                  |
+| `healthIngress.certManager`     | Set this to true in order to add the corresponding annotations for cert-manager                            | `false`                  |
+| `healthIngress.hostname`        | When the heallth ingress is enabled, a host pointing to this will be created                               | `aspnet-core.local`      |
+| `healthIngress.annotations`     | Ingress annotations done as key:value pairs                                                                | `{}`                     |
+| `healthIngress.tls`             | Enable TLS configuration for the hostname defined at ingress.hostname parameter                            | `false`                  |
+| `healthIngress.secrets`         | If you're providing your own certificates, please use this to add the certificates as secrets              | `[]`                     |
+
 
 ### RBAC parameters
 
-| Parameter                               | Description                                                         | Default                                                 |
-|-----------------------------------------|---------------------------------------------------------------------|---------------------------------------------------------|
-| `serviceAccount.create`                 | Enable the creation of a ServiceAccount for ASP.NET Core pods       | `true`                                                  |
-| `serviceAccount.name`                   | Name of the created ServiceAccount                                  | Generated using the `aspnet-core.fullname` template     |
```

Comment:
We are losing these kind of comments saying that the default value is auto generated. Are we ok with that?

Reply:
I think it's ok as long a it is kept in the `values.yaml` like in here:https://github.com/bitnami/charts/blob/6b528440e4737010b736cdcaebbf241bcde2c0e4/bitnami/redis/values.yaml#L999-L1001